cmake/OpenMCConfig.cmake.in

```diff
@@ -40,5 +40,5 @@ if(@OPENMC_USE_MCPL@)
 endif()
 
 if(@OPENMC_USE_UWUW@)
-  find_package(UWUW REQUIRED)
+  find_package(OPEMC_UWUW REQUIRED)
```

Comment:
I don't think this will work, UWUW is a part of DAGMC, so I think, finding DAGMC is enough.

Reply:
Yes. The package name will need to remain the same and not take the value of the compiler definition.

Reply:
Actually, this is not a standalone package that comes with its own config file, so I think this section is not required. Let's see if the tests work. It is a cpp file that comes with DAGMC.

Reply:
This check is already available in CMakelists.txt file. I have added a warning msg and disabled the OpenMC UWUW whenever the DAGMC UWUW is disabled.

Reply:
Oh, actually, the cmake configuration will get fail if DAGMC_BUILD_UWUW is set to OFF or the variable is empty, and get disabled with a warning msg if OPENMC_USE_DAGMC is turned OFF.

Reply:
The `CMakeLists.txt` file will determine the configuration of the OpenMC build itself, but the `CMakeConfig.cmake` file is installed with OpenMC and used by packages that are attempting to build with OpenMC as a dependency. When that occurs, we should ensure that the configuration of the DAGMC package found here matches the requirements of the OpenMC installation (i.e. that DAGMC was built with UWUWU enabled if `OPENMC_USE_UWUW` is true.

Hopefully such a project is using the CMake targets provided by the OpenMC installation instead of this configuration file, which avoids issues like this, but as long as we're providing this option for building and optionally enabling UWUW, it seems best to add this check.